### DISCUSSION TITLE
腾讯视频开屏广告跳过缓慢且有概率不触发点击，已开启shizuku模拟点击

### DISCUSSION THREAD

**Discussion by AIsouler** [gkd-kit/gkd#561] [General]
https://i.gkd.li/i/14648602
```
{
  id: 'com.tencent.qqlive',
  name: '腾讯视频',
  groups: [
    {
      key: 0,
      name: '开屏广告',
      quickFind: true,
      matchTime: 10000,
      actionMaximum: 1,
      resetMatch: 'app',
      rules: [
        {
          key: 0,
          action: 'clickCenter',
          forcedTime: 10000,
          matches: '[text="跳过"]',
          snapshotUrls: 'https://i.gkd.li/i/14648602',
        },
      ],
    },
  ],
}
```

https://github.com/gkd-kit/gkd/assets/57941037/7137dfcb-1803-403b-8945-9b5884b7a11a

**Comment by lisonge** [comment]
跳过缓慢是 腾讯视频 自己的问题，无法解决

无法跳过估计是你的开屏广告规则被其它规则阻塞导致无法执行

**Reply by AIsouler** [reply]
我把其他规则都关了，只开了一个开屏广告，跳过速度上升了，但是还是有概率不触发点击，腾讯视频是否和大麦一样具有某种防御措施？

**Comment by aoguai** [comment]
发现某些场景确实无法跳过，比如支付宝和微信支付后的场景，有时可以有时不行。
特征是：悬浮窗截屏按钮无法在这些场景出现，只能使用音量快照实现截屏

例如：
https://i.gkd.li/i/14650607
https://i.gkd.li/i/14558398

**Comment by lisonge** [comment]
使用此字段可优化被其他规则阻塞的问题

https://gkd.li/api/interfaces/RawCommonProps#prioritytime

**Reply by AIsouler** [reply]
已经能稳定跳过
